Modules/_struct.c

```diff
@@ -497,9 +497,8 @@ nu_ulonglong(_structmodulestate *state, const char *p, const formatdef *f)
 static PyObject *
 nu_bool(_structmodulestate *state, const char *p, const formatdef *f)
 {
-    _Bool x;
-    memcpy(&x, p, sizeof x);
-    return PyBool_FromLong(x != 0);
+    const _Bool _false = false;
+    return PyBool_FromLong(memcmp(p, &_false, sizeof(_Bool)));
```

Comment:
```suggestion
    const _Bool bool_false = false;
    return PyBool_FromLong(memcmp(p, &bool_false, sizeof(_Bool)));
```

Reply:
I'm afraid that names starting with underscore can conflict with an existing name.